metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInstanceProperties.pdl

```diff
@@ -0,0 +1,110 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.CustomProperties
+import com.linkedin.common.ExternalReference
+import com.linkedin.common.Urn
+import com.linkedin.common.JobFlowUrn
+import com.linkedin.common.DataJobUrn
+
+/**
+ * The inputs and outputs of this data process
+ */
+@Aspect = {
+  "name": "dataProcessInstanceProperties",
+  "type": "timeseries",
+}
+record DataProcessInstanceProperties includes CustomProperties, ExternalReference {
+
+  /**
+   * Process name
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
```

Comment:
What is a name of a process? Is this something we usually have?

Reply:
+1 for having a `name` field 😄 

One of the use cases we'd like to use this _DataProcessInstance_ is for modelling kafka stream applications. In Kafka Streams applications there is the APPLICATION_ID https://kafka.apache.org/documentation/streams/developer-guide/config-streams.html#application-id

Another use case could be Spark jobs. There we also have application name https://spark.apache.org/docs/latest/api/scala/org/apache/spark/SparkConf.html#setAppName(name:String):org.apache.spark.SparkConf

Reply:
nice - thanks for the context!